packages/@tinacms/toolkit/src/packages/react-alerts/Alerts.tsx

```diff
@@ -35,27 +43,72 @@ export function Alerts({ alerts }: AlertsProps) {
   }
 
   return (
-    <AlertContainer>
-      {alerts.all.map((alert, i) => {
-        return (
-          <Alert
-            key={alert.id}
-            index={i}
-            level={alert.level}
-            onClick={() => {
-              alerts.dismiss(alert)
-            }}
-          >
-            {alert.level === 'info' && <InfoIcon />}
-            {alert.level === 'success' && <AlertIcon />}
-            {alert.level === 'warn' && <WarningIcon />}
-            {alert.level === 'error' && <ErrorIcon />}
-            <p>{alert.message}</p>
-            <CloseAlert />
-          </Alert>
-        )
-      })}
-    </AlertContainer>
+    <>
+      <AlertContainer>
+        {alerts.all
+          .filter((alert) => {
+            return alert.level !== 'error'
+          })
+          .map((alert, i) => {
+            return (
+              <Alert key={alert.id} index={i} level={alert.level}>
+                {alert.level === 'info' && <InfoIcon />}
+                {alert.level === 'success' && <AlertIcon />}
+                {alert.level === 'warn' && <WarningIcon />}
+                <p>{alert.message}</p>
+                <CloseAlert
+                  onClick={() => {
+                    alerts.dismiss(alert)
+                  }}
+                />
+              </Alert>
+            )
+          })}
+      </AlertContainer>
+      {alerts.all
```

Comment:
I suspect that one of the reasons the previous UI was good was because it handled multiple messages by just stacking them. So for this, what happens when there are 2? And along with that, what about when 1 is an error and another is not? 

For what it's worth, that might be a task for another day,  just want to make sure it's not buggy, if we need to only show a single error message at a time that's probably fine.

Reply:
I think the error modals will stack, with the most recent on top. The alerts display overtop the modal backdrop. I haven't tested with multiple errors, though, so I'm not 100% sure.

Reply:
Confirmed modals stack & other alerts are rendered on top.

<img width="853" alt="Screen Shot 2022-08-29 at 12 10 20 PM" src="https://user-images.githubusercontent.com/5075484/187233798-77fd306b-1977-42d5-a4aa-433f84e790c6.png">